tensorflow_datasets/core/download/downloader.py

```diff
@@ -148,21 +149,39 @@ def _sync_download(self, url, destination_path):
         'https': os.environ.get('TFDS_HTTPS_PROXY', None),
         'ftp': os.environ.get('TFDS_FTP_PROXY', None)
     }
+    CA_BUNDLE = os.environ.get(
+        'TFDS_CA_BUNDLE',
+        None) or os.environ.get(
+        'REQUESTS_CA_BUNDLE',
+        None)
+    if CA_BUNDLE:
+      CA_BUNDLE = extract_zipped_paths(CA_BUNDLE)
+    if not hasattr(ssl, '_create_unverified_context'):
+      # For python version <= 2.7.8
+      ssl.__dict__['_create_unverified_context'] = lambda: None
+      ssl.__dict__['create_default_context'] = lambda *args, **kwargs: None
```

Comment:
In such case - we should be printing a warning inside.

So that users are not surprised that the certificates are not used.

Reply:
Okay. I'm not really sure how to fetch fetch the default logger from tfds. Can you help me out? Like some function or something which returns an object to the logger. I think just `print()`-ing it out is not an option.

Reply:
> from absl import logging
> 
> ssl.**dict**['_create_unverified_context'] = lambda: logging.info("XXXXX")

That looks great! On it!